src/features/speciesfeature.jl

```diff
@@ -24,18 +24,18 @@ end
 
 
 # pretty printing, short version
-Base.show(io::IO, af::AtomFeature) = print(io, "AtomFeature $(af.name)")
+Base.show(io::IO, af::SpeciesFeatureDescriptor) = print(io, "AtomFeature $(af.name)")
 
 # pretty printing, long version
-function Base.show(io::IO, ::MIME"text/plain", af::AtomFeature)
+function Base.show(io::IO, ::MIME"text/plain", af::SpeciesFeatureDescriptor)
```

Comment:
again, we can ditch `contextual` here (I know probably a lot of this came from a rebase onto the incomplete changes I made, but I don't think there's any harm in fixing it here?)

Reply:
Resolved in 43bd2f9.